app/api/odm/specs/ModelWithPermissions.spec.ts

```diff
@@ -18,6 +18,7 @@ describe('ModelWithPermissions', () => {
     permissions: { type: mongoose.Schema.Types.Mixed, select: false },
     fixed: Boolean,
   });
+  const userId = testingDB.id();
```

Comment:
This file used `user1` and `user2` (in line 67 for example).

Probably these are not being used in the test, but maybe it's worth it to create a:
```
const user1Id = testingDB.id();
const user2Id = testingDB.id();
```

and replicate the original structure but removing the deprecated practice of assigning strings to the user?